spark/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala

```diff
@@ -72,8 +72,7 @@ import org.apache.spark.sql.types._
  * A SQL parser that tries to parse Delta commands. If failing to parse the SQL text, it will
  * forward the call to `delegate`.
  */
-class DeltaSqlParser(val delegateSpark: ParserInterface) extends ParserInterfaceShims {
-  private val delegate = ParserInterfaceShims(delegateSpark)
+class DeltaSqlParser(val delegate: ParserInterface) extends ParserInterface {
```

Comment:
Needed because of https://github.com/apache/spark/commit/87917675c4013b4f979c576bc7586efaa3d69a41